features/steps/cocoa_steps.rb

```diff
@@ -1,6 +1,7 @@
 def short_scenario_name(scenario)
   unless scenario.end_with? 'Scenario'
-    raise 'All scenario names must end with "Scenario".  There are removed by the test harness and re-added uniformly by the test fixture.'
+    raise 'All scenario names must end with "Scenario".  ' \
+          'They are removed by the test harness and re-added uniformly by the test fixture.'
```

Comment:
To me this looks uglier / less readable when split onto two lines with `\`

Maybe it doesn't need to be so verbose?

Also there's a typo - "There are removed ..." instead of "These are removed ..."

Reply:
I wrapped it in anticipation of Rubocop complaining about line lengths at some point in the future, but have now shortened the message and added an explanation as a comment - to try and avoid someone taking it out one day.